sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -381,6 +389,11 @@ export interface QueueProperties {
    * `sb://<your-service-bus-namespace-endpoint>/<queue-or-topic-name>`
    */
   forwardDeadLetteredMessagesTo?: string;
+
+  /**
+   * Specifies whether express entities are enabled on queue.
+   */
+  enableExpress: boolean;
```

Comment:
This is not a property that can be updated after a queue is created. We have been following the pattern of marking such properties as `readonly` (see enablePartitioning a few lines above)

```suggestion
  readonly enableExpress: boolean;
```